src/statsmodels/statsmodel.jl

```diff
@@ -48,8 +48,8 @@ for (modeltype, dfmodeltype) in ((:StatisticalModel, DataFrameStatisticalModel),
                                  (:RegressionModel, DataFrameRegressionModel))
     @eval begin
         function StatsBase.fit{T<:$modeltype}(::Type{T}, f::Formula, df::AbstractDataFrame,
-                                              args...; kwargs...)
-            mf = ModelFrame(f, df)
+                                              args...; contrasts::Dict = Dict(), kwargs...)
```

Comment:
In general I think the recommended pattern in Julia is to pass instances rather than types, which is e.g. how specifying a distribution works. So if it's just to avoid typing `()`, I'd rather not support this variant to keep things simple. We can always add it later if it really turns out to be useful.